cgo/cgo.go

```diff
@@ -210,13 +215,13 @@ func Process(files []*ast.File, dir string, fset *token.FileSet, cflags []string
 		}
 	}
 	// Patch some types, for example *C.char in C.CString.
-	cf := p.newCGoFile()
+	cf := p.newCGoFile(nil, -1)
```

Comment:
Is there some special significance to the `-1` and if so perhaps a named `const` instead?

Reply:
No, not really. I'm just creating a dummy `*cgoFile` object. I have added a comment that should hopefully make this clearer.